.github/workflows/integration.yml

```diff
@@ -223,8 +224,10 @@ jobs:
             runner-image: ubuntu-20.04
           - install-mode: pip-default
             pip-install-target: '.'
-          - install-mode: pip-optional
-            pip-install-target: .[optional]
+          - install-mode: pip-complete
+            pip-install-target: .[complete]
```

Comment:
Are these still valid options? I seem to recall you deciding to drop the "complete" and "all" options, and I do not see them later in setup.py.

Reply:
You're absolutely correct, thanks for spotting that. I'll modify the workflow files to account for this.